liquibase-standard/src/main/java/liquibase/util/StringUtil.java

```diff
@@ -1171,4 +1172,13 @@ public static List<String> splitToChunks(String input, int chunkSize) {
         }
         return chunks;
     }
+
+    /**
+     *
+     * Remove the "file://" prefix from a URI if it exists.
+     *
+     */
+    public static String stripUriPrefix(URI uri) {
```

Comment:
This method name can be misleading, could it be `stripFileUrlPrefix` or something like that?

Reply:
Good call! I've renamed this to `stripFileUriPrefix`. I've also migrated this out of StringUtil since the only usage was in a single InitProject class. It can stay private for now.